src/buildblock/ProjData.cxx

```diff
@@ -235,32 +235,22 @@ ProjData::get_subset(const std::vector<int>& views) const
 {
   auto subset_proj_data_info_sptr =
     std::make_shared<ProjDataInfoSubsetByView>(proj_data_info_sptr, views);
-  auto subset_proj_data_ptr = new ProjDataInMemory(exam_info_sptr, subset_proj_data_info_sptr);
-  auto& subset_proj_data = *subset_proj_data_ptr;
+  unique_ptr<ProjDataInMemory> subset_proj_data_uptr(new ProjDataInMemory(exam_info_sptr, subset_proj_data_info_sptr));
 
+  //TODOTOF loop here
```

Comment:
Is this TODO needed before merge?

Reply:
no. I put it in such that we don't forget when merging to the TOF branch